src/main/resources/templates/maintenance.html

```diff
@@ -0,0 +1,26 @@
+<page th:replace="fragments/maintenance-layout :: page(~{::title},~{::content})">
```

Comment:
not sure if the contents of maintenance page have been approved by the client. Also if we can also provide info about when the site will be available?

Reply:
good question - I believe I took the content from what was already used as maintenance page - can be changed if needs be of course